client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/upgrade-modal.scss

```diff
@@ -169,6 +169,8 @@ $design-button-primary-hover-color: var(--color-primary-60);
 			fill: var(--studio-green-50);
 			flex-shrink: 0;
 			margin-right: 10px;
+			align-self: flex-start;
```

Comment:
This works for me, although Chrome tells me that `align-self` doesn't work for `block` elements, as [confirmed by MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/align-self#:~:text=The%20property%20doesn%27t%20apply%20to%20block%2Dlevel%20boxes%2C%20or%20to%20table%20cells.%20If%20a%20flexbox%20item%27s%20cross%2Daxis%20margin%20is%20auto%2C%20then%20align%2Dself%20is%20ignored.).

This said, it's undeniable that `align-self` _is_ doing something here. 😄 

I've tried removing it and [applying it to `.upgrade-modal__included li`](https://github.com/Automattic/wp-calypso/blob/19c36dd5c7476863090b9efd81667a035b08794e/client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/upgrade-modal.scss#L163-L166) and I got the same result without Chrome being all nitpicky.

```diff
 li {
   display: flex;
-  align-items: center;
+  align-items: flex-start;
 }
```

Reply:
Oh thanks, much better!